### ISSUE TITLE
Add Comment should be disabled when the commenting box is empty

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode-pull-request-github#128]
Testing https://github.com/Microsoft/vscode/issues/55396.

Keyboard focus is in the commenting box. Since the comment is empty, Add Comment should be disabled.

![image](https://user-images.githubusercontent.com/4674940/43533858-b801294c-956a-11e8-8956-cb02421ac804.png)
